### PR TITLE
Ctrl -> Command in macOS article

### LINKED ISSUE #14290: Incorrect hot keys used
Since this is the macOS version of getting started a few changes need to happen.

Under the header Debug the Application, first paragraph it says to use F9 to insert a breakpoint.  On a mac users must use the combination fn+F9.  In the second paragraph, the hotkey combo listed to show the debug screen is Ctrl + Shift + D.  On a mac, it should be Cmd + Shift + D.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3343b35c-90c1-2e11-498d-ee1d4eb409f6
* Version Independent ID: f2eda261-d4c3-c90a-1294-dfe11e55c341
* Content: [Get started with .NET Core on macOS - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-on-macos)
* Content Source: [docs/core/tutorials/using-on-macos.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/using-on-macos.md)
* Product: **dotnet-core**
* GitHub Login: @bleroy
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #14290

### CODE REVIEW

**Comment by mairaw** [comment]
We should probably switch to the keys shown in the VS Code docs (see the macOS one):
image

**Comment by Youssef1313** [comment]
@mairaw, Do you mean swapping the `command` ( &#8984; ) with the `shift` ? or adding the `shift` symbol ?

File: docs/core/tutorials/using-on-macos.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ dotnet run -p app/app.csproj
 
 Set a breakpoint at the `WriteLine` statement in the `Main` method. Do this by either pressing the <kbd>F9</kbd> key when the cursor is over the `WriteLine` line or by clicking the mouse in the left margin on the line where you want to set the breakpoint. A red circle will appear in the margin next to the line of code. When the breakpoint is reached, code execution will stop *before* the breakpoint line is executed.
 
-Open the debugger tab by selecting the Debug icon in the Visual Studio Code toolbar, selecting **View > Debug** from the menu bar, or using the keyboard shortcut <kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>D</kbd>:
+Open the debugger tab by selecting the Debug icon in the Visual Studio Code toolbar, selecting **View > Debug** from the menu bar, or using the keyboard shortcut <kbd>&#8984;</kbd>+<kbd>SHIFT</kbd>+<kbd>D</kbd>:
```
Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: I'd match what they do in the VS Code docs

```suggestion
Open the debugger tab by selecting the Debug icon in the Visual Studio Code toolbar, selecting **View > Debug** from the menu bar, or using the keyboard shortcut <kbd>&#8679;</kbd><kbd>&#8984;</kbd><kbd>D</kbd>:
```

**Review by mairaw — CHANGES_REQUESTED** [review]

**Comment by Youssef1313** [comment]
I'll update the shortcut key for integrated terminal to match VS Code docs too.

image

**Comment by Youssef1313** [comment]
@mairaw, I've made some other changes. Can you review them ?

**Comment by Youssef1313** [comment]
Ping @mairaw.

**Comment by Youssef1313** [comment]
Ping @mairaw

**Comment by mairaw** [comment]
This article has been reviewed again since then. Can you resolve the merge conflicts?

File: docs/core/tutorials/using-on-macos.md [hunk 2]
```diff
@@ -205,7 +206,7 @@ dotnet run -p app/app.csproj
 
 Set a breakpoint at the `WriteLine` statement in the `Main` method. Do this by either pressing the <kbd>Fn</kbd>+<kbd>F9</kbd> key when the cursor is over the `WriteLine` line or by clicking the mouse in the left margin on the line where you want to set the breakpoint. A red circle will appear in the margin next to the line of code. When the breakpoint is reached, code execution will stop *before* the breakpoint line is executed.
 
-Open the debugger tab by selecting the Debug icon in the Visual Studio Code toolbar, selecting **View > Debug** from the menu bar, or using the keyboard shortcut <kbd>Command</kbd>+<kbd>SHIFT</kbd>+<kbd>D</kbd>:
+Open the debugger tab by selecting the Debug icon in the Visual Studio Code toolbar, selecting **View** > **Debug** from the menu bar, or using the keyboard shortcut <kbd>&#8679;</kbd><kbd>&#8984;</kbd><kbd>D</kbd>:
```
Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: On our style guide, I always see keys represented with the + sign so I think we should keep that

Comment (mairaw) [inline docs/core/tutorials/using-on-macos.md]: @BillWagner can you just validate this change for the keyboard keys? Everything else looks good to me.

Comment (BillWagner) [reply docs/core/tutorials/using-on-macos.md]: @mairaw Yes, that's correct. Command + SHIFT + D.  (or ⌘⇧ D)

**Review by mairaw — APPROVED** [review]

**Pull request merged**